cmd/up/up.go

```diff
@@ -274,6 +274,9 @@ func Up() *cobra.Command {
 					return err
 				}
 			}
+			if upOptions.commandToExecute != nil {
```

Comment:
We should handle empty slices too
```suggestion
			if len(upOptions.commandToExecute) > 0 {
```

Reply:
Won't an uninitialized `nil` slice have length 0 as well?